test/inspector/test-inspector-ip-detection.js

```diff
@@ -22,7 +22,7 @@ function checkListResponse(instance, err, response) {
   const match = wsUrl.match(/^ws:\/\/(.*):9229\/(.*)/);
   assert.strictEqual(ip, match[1]);
   assert.strictEqual(res['id'], match[2]);
-  assert.strictEqual(ip, res['devtoolsFrontendUrl'].match(/.*ws=(.*):9229/)[1]);
+  assert.strictEqual(ip, res['devtoolsFrontendUrl'].match(/ws=(.*):9229/)[1]);
```

Comment:
I would leave it as is. If there's no good reason to make the test more lax, don't make it more lax. :-D

If the intention is to introduce a lint rule that would flag something like this line, it can always be disabled on just this line with a comment.